src/main/features/dashboard/views/macro/claimStatus/statesPaidRejected.njk

```diff
@@ -56,13 +56,19 @@
   {% endif %}
   {% set mediation = claim.claimantResponse.freeMediation and claim.claimantResponse.freeMediation === YesNoOption.YES.option %}
 
+  <h2 class="heading-small">{{ t('Wait for the court to review the case') }}</h2>
   {% if mediation %}
-    <h2 class="heading-small">{{ t('We’ll contact you with a mediation appointment') }}</h2>
+    <p>{{ t('We’ll contact you with a mediation appointment') }}</p>
+  {% else %}
+    <p>{{ t('{{ claimantName }} has rejected your admission of {{ amount }}', { claimantName: claim.claimData.claimant.name, amount: amount | numeral}) }}<p>
+  {% endif %}
+
+  {% if claim.claimantResponse.paymentReceived === YesNoOption.YES.option %}
+    <p>{{ t('They accept that you’ve paid them {{ partAmount }}, but believe you owe the full {{ fullAmount}} claimed.', { partAmount: claim.response.amount | numeral , fullAmount: claim.totalAmountTillToday | numeral }) }}</p>
```

Comment:
Can you check this content is tested somewhere?

Reply:
done